_data/JasonSmitR/JasonSmitR.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Jason Ronald Smit
+institution: The University of The Witwatersrand
+profile_pic: jason.png
+quote: I took the senic route but I still arrived
```

Comment:
Typo? Change `senic` to `scenic`.

Reply:
Thanks Mohammed-Shoaib for the spot! I made a new PR to resolve the issue since this one got merged.